bin/local-test

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+export PATH="node_modules/.bin:$PATH"
+export ALLOW_CONFIG_MUTATIONS=true
+export NODE_ENV=test
+export TZ=UTC
+
+# Run Standard
+standard &&
+
+# Run tests
+mocha -R list -r must -r reify --exit
```

Comment:
I noticed the `bin/` folder seems to vary between this project and `conflator`. This bin folder has `local-test` and `local-coverage` while conflator also includes `ci` and `check-ci`. What are the standard bash files we need to add to all of these repos in the `bin` directory @qjflores @rlpacht (if we can figure this out here, that'd be great 👍 
 so that we're on the same page) ?

Reply:
`local-test` and `local-coverage` we can delete `ci` and `check-ci` from repos